test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -649,3 +633,185 @@
+#   - opcode: createPods
+#     namespace: test
+#     countParam: $measurePods
+#     podTemplatePath: config/pod-with-claim-reference.yaml
+#     collectMetrics: true
+#   workloads:
+#   - name: base
+#     params:
+#       nodesWithDRA: 1
+#       nodesWithoutDRA: 1
+#       initPods: 0
+#       measurePods: 1
+#       maxClaimsPerNode: 10
+#   - name: 1node
+#     params:
+#       nodesWithDRA: 1
+#       nodesWithoutDRA: 0
+#       initPods: 0
+#       measurePods: 100
+#       maxClaimsPerNode: 100
+#   - name: 10nodes
+#     params:
+#       nodesWithDRA: 10
+#       nodesWithoutDRA: 0
+#       initPods: 0
+#       measurePods: 100
+#       maxClaimsPerNode: 10
+#   - name: 2000pods_200nodes
+#     params:
+#       nodesWithDRA: 200
+#       nodesWithoutDRA: 0
+#       initPods: 1000
+#       measurePods: 1000
+#       maxClaimsPerNode: 10
+#   - name: 2000pods_100nodes
+#     params:
+#       nodesWithDRA: 100
+#       nodesWithoutDRA: 0
+#       initPods: 1000
+#       measurePods: 1000
+#       maxClaimsPerNode: 20
+
+- name: SchedulingWithInlineResourceClaims
+  featureGates:
+    DynamicResourceAllocation: true
+  workloadTemplate:
+  - opcode: createNodes
+    countParam: $nodesWithoutDRA
+  - opcode: createNodes
+    nodeTemplatePath: config/node-with-dra-test-driver.yaml
+    countParam: $nodesWithDRA
+  - opcode: createResourceDriver
+    driverName: test-driver.cdi.k8s.io
+    nodes: scheduler-perf-dra-*
+    maxClaimsPerNodeParam: $maxClaimsPerNode
+  - opcode: createResourceClass
+    templatePath: config/resourceclass.yaml
+  - opcode: createResourceClaimTemplate
+    templatePath: config/resourceclaimtemplate.yaml
+    namespace: init
+  - opcode: createPods
+    namespace: init
+    countParam: $initPods
+    podTemplatePath: config/pod-with-claim-template.yaml
+  - opcode: createResourceClaimTemplate
+    templatePath: config/resourceclaimtemplate.yaml
+    namespace: test
+  - opcode: createPods
+    namespace: test
+    countParam: $measurePods
+    podTemplatePath: config/pod-with-claim-template.yaml
+    collectMetrics: true
+  workloads:
+  - name: fast
+    params:
+      # This testcase runs through all code paths without
+      # taking too long overall.
+      nodesWithDRA: 1
+      nodesWithoutDRA: 1
+      initPods: 0
+      measurePods: 10
+      maxClaimsPerNode: 10
+  - name: 2000pods_100nodes
+    params:
+      # In this testcase, the number of nodes is smaller
+      # than the limit for the PodScheduling slices.
+      nodesWithDRA: 100
+      nodesWithoutDRA: 0
+      initPods: 1000
+      measurePods: 1000
+      maxClaimsPerNode: 20
+  - name: 2000pods_200nodes
+    params:
+      # In this testcase, the driver and scheduler must
+      # truncate the PotentialNodes and UnsuitableNodes
+      # slices.
+      nodesWithDRA: 200
+      nodesWithoutDRA: 0
+      initPods: 1000
+      measurePods: 1000
+      maxClaimsPerNode: 10
+
+- name: LatencyWithInlineResourceClaims
```

Comment:
why is this called "Latency"? Add a comment to explain the difference with the previous test.

Reply:
Added:

> This test case schedules one pod after the other and thus is dominated by the
> time that it takes to complete the scheduling of a single pod. In the other
> benchmarks, parallelism can hide the effect of delays per pod.